cogs/tips.py

```diff
@@ -2,7 +2,8 @@
 
 from typing import TYPE_CHECKING
 
-from discord import Member, option
+from discord import ApplicationContext, Member, User, option
```

Comment:
`ApplicationContext` is redefined here? It appears twice now

Reply:
Oops lol, rebase issue. Removed double import in 55949e2b6cdc74af7327d949f3dd2f8693cdffab